### ISSUE TITLE
ASSERTION ERROR:sentence at index 8241 of size 6922994 exceeds max_tokens limit of 1280000!

### ISSUE THREAD

**Issue by MorrisXu-Driving** [pytorch/fairseq#2561]
#### What is your question?
I am trying to fine-tune the model using my own audio and label. However, I met this ASSERTION ERROR:**sentence at index 8241 of size 6922994 exceeds max_tokens limit of 1280000!**  Does this mean that there is something in label file not included in dict.txt.ltr? How to solve it?
#### Code
nohup python /data/home/v_liwenxu/fairseq/fairseq_cli/train.py //data3/mirex2020/label \
--save-dir /data/home/v_liwenxu/data/segmented_sing/s5/models/ --fp16 \
--post-process letter --valid-subset //data3/mirex2020/label/valid \
--no-epoch-checkpoints --best-checkpoint-metric wer --num-workers 4 \
--max-update 80000 --sentence-avg --task audio_pretraining --arch wav2vec_ctc --w2v-path /data/home/v_liwenxu/data/segmented_sing/s5/models/wav2vec_small.pt \
--labels ltr --apply-mask --mask-selection static --mask-other 0 --mask-length 10 --mask-prob 0.5 --layerdrop 0.1 \
--mask-channel-selection static --mask-channel-other 0 --mask-channel-length 64 --mask-channel-prob 0.5 --zero-infinity \
--feature-grad-mult 0.0 --freeze-finetune-updates 10000 --validate-after-updates 10000 --optimizer adam \
--adam-betas '(0.9, 0.98)' --adam-eps 1e-08 --lr 2e-05 --lr-scheduler tri_stage --warmup-steps 8000 --hold-steps 32000 \
--decay-steps 40000 --final-lr-scale 0.05 --final-dropout 0.0 --dropout 0.0 --activation-dropout 0.1 --criterion ctc \
--attention-dropout 0.0 --max-tokens 1280000 --seed 2337 --log-format json --log-interval 500  --ddp-backend no_c10d  > train_output.txt 2>&1 &
####

**Comment by MorrisXu-Driving** [comment]
I just need to adjust the max-tokens in the training script.

**Comment by ArtemisZGL** [comment]
@MorrisXu-Driving hello, i met this error too. If you just adjust the max_tokens, will it cause the oom error?

**Comment by MorrisXu-Driving** [comment]
I actually don't know what is the real reason that triggered this error. I just used a smaller dataset (nearly 90%) than the one that triggered this error and everything started working propearly. I guess this may because of some unexpected tokens in labels of your original dataset.

**Comment by ArtemisZGL** [comment]
@MorrisXu-Driving oh, thanks. I just filter some sample which is longer than max tokens. It seems that some sample is strange.
